autoload/neomake/makers/ft/rust.vim

```diff
@@ -123,6 +123,7 @@ endfunction
 " elements per line.
 function! neomake#makers#ft#rust#CargoProcessOutput(context) abort
     let errors = []
+    let cwd = get(get(get(a:context, 'jobinfo', {}), 'maker'), 'cwd', '')
```

Comment:
I (would) assume that this should be there always.. so just access it directly - it being empty is probably something unexpected further down anyway, isn't it?

Reply:
fixed